datafusion/execution/src/task.rs

```diff
@@ -92,13 +100,15 @@ impl TaskContext {
             config.set(&k, &v)?;
         }
         let session_config = SessionConfig::from(config);
+        let window_functions = HashMap::new();
```

Comment:
I guess, since this API is deprecated anyway, you didn't update its arguments.

Reply:
Yes, exactly. I also tried to lead people to `SessionContext::task_ctx` with another doc comment above.